scripts/DickeModel.py

```diff
@@ -55,28 +55,35 @@ def main(args):
     print('starting')
     value_per_circuit = value/repetitions
     value_per_circuit=6
-    dicke_metadata = EstimateMetaData(
+    #TODO: See if I need to refactor the size string to include the variable names
+    dicke_metadata = GSEEMetaData(
         id=time.time_ns(),
         name=name,
         category='scientific',
         size=f'{n_b} + 1 + {n_s}',
         task='Ground State Energy Estimation',
-        implementations=f'GSEE, evolution_time={t_dicke}, bits_precision={bits_precision_dicke}, trotter_order={trotter_order_dicke}, n_s={n_s}, n_b={n_b}',
         value_per_circuit=value_per_circuit,
-        repetitions_per_application=repetitions
+        repetitions_per_application=repetitions,
+
+        
+        evolution_time=t_dicke,
+        trotter_order = trotter_order_dicke,
+        bits_precision = bits_precision_dicke,
+        nsteps=trotter_steps_dicke,
     )
 
     print('Estimating Dicke')
     t0 = time.perf_counter()
     estimate = gsee_resource_estimation(
         outdir=directory,
-        numsteps=trotter_steps_dicke,
+        nsteps=trotter_steps_dicke,
         gsee_args=args_dicke,
         init_state=init_state_dicke,
         precision_order=1, #actual precision bits accounted as scaling factors in the resource estimate
         phase_offset=dicke_phase_offset,
         bits_precision=bits_precision_dicke,
         circuit_name=name,
+        is_extrapolated = is_extrapolated,
```

Comment:
is_extrapolated is not defined above, I see in other scripts, it is defined as 'is_extrapolated = args.extrapolate'